src/g_array.c

```diff
@@ -806,12 +808,10 @@ static void garray_save(t_gobj *z, t_binbuf *b)
     garray_savecontentsto(x, b);
 
     if (pd_compatibilitylevel >= 52) {
-        t_float fval;
-        style = template_getfloat(
+        t_symbol *color = template_getsymbol(
             scalartemplate, gensym("color"), x->x_scalar->sc_vec, 1);
-        binbuf_addv(b, "ssi;", gensym("#A"), gensym("color"), style);
-
-        fval = template_getfloat(
+        binbuf_addv(b, "sss;", gensym("#A"), gensym("color"), color);
+        t_float fval = template_getfloat(
             scalartemplate, gensym("linewidth"), x->x_scalar->sc_vec, 1);
```

Comment:
this  breaks compatibility with older Pds.

Reply:
I see. I guess you're the one to come up with a good solution. How was it for iemguis when it started saving colors as symbols? 

Could we get the symbol and save it as an int? That wouldn't give errors but it would open the patch on an older version with the wrong color. So I don't know...